delivery_carrier_deposit/wizard/deposit.py

```diff
@@ -40,23 +40,33 @@ def _get_carrier_type_selection(self):
         required=True, help="Carrier type (combines several delivery "
         "methods). Make sure that the option 'Deposit Slip' is checked on "
         "the delivery methods that have this carrier type.")
+    picking_type_id = fields.Many2one(
+        comodel_name='stock.picking.type',
+        string='Picking Type',
+        required=True)
 
     @api.model
     def _prepare_deposit_slip(self):
         return {
             'carrier_type': self.carrier_type,
             'company_id': self.env.user.company_id.id,
+            'picking_type_id': self.picking_type_id.id,
             }
 
+    @api.model
+    def _get_deposit_pickings(self):
+        return self.env['stock.picking'].search([
```

Comment:
_get_deposit_domain() could be more easy to override by 2 other modules with append() than this method